src/servers/main.cc

```diff
@@ -94,6 +94,12 @@ bool grpc_use_ssl_ = false;
 nvidia::inferenceserver::SslOptions grpc_ssl_options_;
 grpc_compression_level grpc_response_compression_level_ =
     GRPC_COMPRESS_LEVEL_NONE;
+int32_t grpc_arg_keepalive_time_ms_ = 7200000;
```

Comment:
Can you add a comment here saying the defaults were picked from https://grpc.github.io/grpc/cpp/md_doc_keepalive.html ?